src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -344,4 +354,44 @@ namespace winrt::TerminalApp::implementation
 
         return bindingsArray;
     }
+    Windows::System::VirtualKeyModifiers AppKeyBindings::ConvertVKModifiers(Settings::KeyModifiers modifiers)
+    {
+        Windows::System::VirtualKeyModifiers keyModifiers = Windows::System::VirtualKeyModifiers::None;
+        
+        if (WI_IsFlagSet(modifiers, Settings::KeyModifiers::Ctrl))
+        {
+            keyModifiers |= Windows::System::VirtualKeyModifiers::Control;
+        }
+        if (WI_IsFlagSet(modifiers, Settings::KeyModifiers::Shift))
+        {
+            keyModifiers |= Windows::System::VirtualKeyModifiers::Shift;
+        }
+        if (WI_IsFlagSet(modifiers, Settings::KeyModifiers::Alt))
+        {
+            // note: Menu is the Alt VK_MENU
+            keyModifiers |= Windows::System::VirtualKeyModifiers::Menu;
+        }
+
+        return keyModifiers;
+    }
+
+    winrt::hstring AppKeyBindings::FormatOverrideShortcutText(Settings::KeyModifiers modifiers)
+    {
+        std::wstring buffer{ L"" };
+
+        if (WI_IsFlagSet(modifiers, Settings::KeyModifiers::Ctrl))
+        {
+            buffer += L"Ctrl+";

```

Comment:
we'll need to localize these at some point; can you tag them with `#794`?

Reply:
As a point on this one, if there yields a better fix to the XAML framework, it will automatically get the localization.  But yes will tag!